cdp_scrapers/scraper_utils.py

```diff
@@ -187,42 +186,8 @@ def parse_static_file(file_path: Path) -> ScraperStaticData:
     -----
     Function looks for "seats", "primary_bodies", "persons" top-level keys
     """
-    with open(file_path) as static_file:
-        static_json: Dict[str, Dict[str, Any]] = json.load(static_file)
-
-        if "seats" not in static_json:
-            seats: Dict[str, Seat] = {}
-        else:
-            seats: Dict[str, Seat] = {
-                seat_name: Seat.from_dict(seat)
-                for seat_name, seat in static_json["seats"].items()
-            }
-
-        if "primary_bodies" not in static_json:
-            primary_bodies: Dict[str, Body] = {}
-        else:
-            primary_bodies: Dict[str, Body] = {
-                body_name: Body.from_dict(body)
-                for body_name, body in static_json["primary_bodies"].items()
-            }
-
-        if "persons" not in static_json:
-            known_persons: Dict[str, Person] = {}
-        else:
-            known_persons: Dict[str, Person] = {
-                person_name: parse_static_person(person, seats, primary_bodies)
-                for person_name, person in static_json["persons"].items()
-            }
-
-        log.debug(
-            f"ScraperStaticData parsed from {file_path}:\n"
-            f"    seats: {list(seats.keys())}\n"
-            f"    primary_bodies: {list(primary_bodies.keys())}\n"
-            f"    persons: {list(known_persons.keys())}\n"
-        )
-        return ScraperStaticData(
-            seats=seats, primary_bodies=primary_bodies, persons=known_persons
-        )
+    with open(file_path, "r") as static_file:
+        return ScraperStaticData.from_json(static_file.read())
```

Comment:
This is awesome. Thank you so much. I do want to compare the returned data before/after. I can't remember off-memory if we have tests for `parse_static_file`. I think we do by proxy, via tests of event gathering (which would use data contained in these files).
Anyway, let me check this before finally merging; though the code should be even more sane now, since we are now converting from JSON as-is.

Reply:
I usually like to do cleanup activities in a separate PR as they tend to be significant events that warrant their own discussions. We can group up the cleanup of multiple unused functions there.